### ISSUE TITLE
Some question about the training

### ISSUE THREAD

**Issue by slnxyr** [zhiweihu1103/HKGC-HyperFormer#1]
I run the run.sh, but the loss seems remains to be unchanged. Is there something wrong in settings?
![image](https://github.com/zhiweihu1103/HKGC-HyperFormer/assets/53633782/8ab5ea65-6d08-4f89-bb5d-75af7003408c)

**Comment by zhiweihu1103** [comment]
Sorry, just saw it. According to experience, there are two reasons that may cause this problem,
1. Regarding the pytorch version, we are using 1.8.1. If you are using pytorch 2, you may need to lower the version;
2. Regarding fastmoe, are you using the fastmoe I provided in the Readme (https://drive.google.com/file/d/1c3ijOe5PacVWyfmD2amUk0dpTbsjIHIS/view?usp=sharing)? Because someone has encountered something similar before.
